package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "loopback-workspace",
   "version": "4.0.0-alpha.1",
-  "main": "index.js",
+  "main": "server/server.js",
+  "engines": {
+    "node": ">=4"
+  },
   "scripts": {
-    "test": "node .",
-    "lint": "eslint ."
+    "test": "eslint .",
+    "lint": "eslint .",
+    "posttest": "npm run lint"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/strongloop/loopback-workspace.git"
   },
   "license": "MIT",
   "dependencies": {
-    "loopback": "^3.0.0"
+    "chai": "^3.5.0",
+    "cucumber": "^1.3.1",
+    "loopback": "^3.0.0",
+    "loopback-boot": "^2.23.0",
+    "loopback-component-explorer": "^2.7.0"
```

Comment:
Why do we have a dep on boot and explorer (and if they are required, why not the latest versions 3.x for boot, 4.x for explorer)?